examples/grid.cpp

```diff
@@ -54,9 +55,31 @@ extern "C" {
 #include "common/common_def.h"
 #include "common/condition.h"
 #include "common/lock.h"
+#include "font.h"
 
 using namespace std;
 
+class Timer
+{
+public:
+    Timer(const char* str):m_str(str)
```

Comment:
It's intended to. we do not want wast time to construct string. Since we always send const char* to constructor.